test/acceptance/features/verifyExamples.feature

```diff
@@ -0,0 +1,73 @@
+Feature: Verify examples provided in Service Binding Operator github repository
```

Comment:
I would vote for naming this file simply `examples.feature`

Reply:
Done,